fidelisnetwork.json

```diff
@@ -0,0 +1,1073 @@
+            "type": "investigate",
+            "identifier": "list_alerts",
+            "read_only": true,
+            "parameters": {
+                "column": {
+                    "description": "Column for ordering",
+                    "data_type": "string",
+                    "required": true,
+                    "default": "ALERT_TIME",
+                    "value_list": [
+                        "ALERT_ID",
+                        "ALERT_TIME"
+                    ],
+                    "order": 0
+                },
+                "direction": {
+                    "description": "Direction of alerts order",
+                    "default": "DESC",
+                    "required": true,
+                    "value_list": [
+                        "DESC",
+                        "ASC"
+                    ],
+                    "data_type": "string",
+                    "order": 1
+                },
+                "start_time": {
+                    "description": "Start time in UTC (YYYY-MM-DD HH:MM:SS)",
+                    "data_type": "string",
+                    "order": 2
+                },
+                "end_time": {
+                    "description": "End time in UTC (YYYY-MM-DD HH:MM:SS)",
+                    "data_type": "string",
+                    "order": 3
+                },
+                "limit": {
+                    "description": "Specify the maximum number of alerts to return. You can specify between 1 and 200,000. (Default is 100)",
+                    "data_type": "numeric",
+                    "default": 100,
+                    "order": 4
+                }
+            },
+            "render": {
+                "type": "table",
+                "width": 12,
+                "height": 5,
+                "title": "List Alerts"
+            },
+            "output": [
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "example_values": [
+                        "success",
+                        "failed"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.total_alerts",
+                    "example_values": [
+                        512
+                    ],
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "action_result.data.*.aaData.*.ALERT_ID",
+                    "example_values": [
+                        false
+                    ],
+                    "column_name": "Alert ID",
+                    "column_order": 0,
+                    "data_type": "string",
+                    "contains": [
+                        "alert id"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.aaData.*.SEVERITY",
+                    "example_values": [
+                        false
```

Comment:
Should the example value here be SEVERITY word? 
I'm surprised to see Boolean value here. The same apply to the `false` value below:

Reply:
Yes, The example value should be value of severity. I updated it.